src/Razor/src/Microsoft.AspNetCore.Razor.Language/DefaultBoundAttributeDescriptorBuilder.cs

```diff
@@ -82,6 +97,18 @@ public BoundAttributeDescriptor Build()
                 diagnostics.UnionWith(_diagnostics);
             }
 
+            var parameters = Array.Empty<BoundAttributeParameterDescriptor>();
+            if (_attributeParameterBuilders != null)
+            {
+                var parameterset = new HashSet<BoundAttributeParameterDescriptor>(BoundAttributeParameterDescriptorComparer.Default);
```

Comment:
@rynowak is the intent for parameters to be case insensitive?

Reply:
parameters will be case-sensitive